frameworks/elastic/src/main/dist/install-plugins.sh

```diff
@@ -20,7 +20,7 @@ if [ "$XPACK_ENABLED" = true ]; then
 fi
 
 if [ -n "$STATSD_UDP_HOST" ]; then
-    STATSD_PLUGIN="file://$MESOS_SANDBOX/elasticsearch-statsd-$ELASTIC_STATSD_VERSION.zip"
+    STATSD_PLUGIN="file://$MESOS_SANDBOX/elasticsearch-statsd-$ELASTIC_VERSION.0.zip"
```

Comment:
This is another thing that we could consider leaving in (or adding as an additional commit on top of the revert)?